### ISSUE TITLE
Complete Node.js + hem/Spine example

### ISSUE THREAD

**Issue by JoostvanPinxten** [baconjs/bacon.js#209]
Is it possible that the current version that is installed through npm (i'm using node.js v0.10.12 to include Bacon) does not work properly? I have tried including it in several ways, but the Bacon module just does not get loaded. Even though the package.json is loaded, it does not (properly?) export the Bacon!

So require(baconjs).Bacon fails to load the module. No other message whatsoever, obviously.

What I've done:
- npm install baconjs
- edited the slug.json modules to include the 'baconjs' module
- restarted the server (i'm using hem for a Spine app)

If I do it like this, then the application.js DOES have the Bacon! Unfortunately, it simply cannot load it (nor any other variant of Bacon, bacon, baconjs etc).

So, basically, what I would like to see is a complete Bacon example on node.js, as it seems that I'm missing something (probably silly) here.

I have been able to load Bacon through loading the Bacon.js script directly, but then it doesn't create the asEventStream, due to jQuery being loaded later than Bacon.
